Default data directory changed to /var/lib/netxms
--- a/src/agent/core/nxagentd.h
+++ b/src/agent/core/nxagentd.h
@@ -80,7 +80,7 @@
 #define AGENT_DEFAULT_CONFIG_D   _T("{search}")
 #define AGENT_DEFAULT_LOG        _T("/var/log/nxagentd")
 #define AGENT_DEFAULT_FILE_STORE _T("/tmp")
-#define AGENT_DEFAULT_DATA_DIR   _T("{default}")
+#define AGENT_DEFAULT_DATA_DIR   _T("/var/lib/netxms")
 #endif
 
 #define REGISTRY_FILE_NAME       _T("registry.dat")
